integration/README.md

```diff
@@ -1,5 +1,17 @@
 # Integration Tests
 
+## Running individual tests in Intellij
```

Comment:
do you want to add the terminal test running instructions? or is that elsewhere?

Reply:
sure, i'll add that